README.md

```diff
@@ -22,7 +23,7 @@ Transparently use [Twitter Bootstrap 3][twbs3] with [CakePHP 3][cakephp].
 - Sample layouts (cover, signin, dashboard)
 - Bake templates *incomplete*
 
-## Install
+## Installing BootstrapUI via Composer
```

Comment:
Maybe better `## Installation` for consistency with other FoC repos (i.e. Crud, CrudView)